buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -855,9 +855,10 @@ class BeamModulePlugin implements Plugin<Project> {
       // configurations because they are never required to be shaded or become a
       // dependency of the output.
       def compileOnlyAnnotationDeps = [
-        "com.google.auto.value:auto-value-annotations:1.7",
+        "com.google.auto.value:auto-value-annotations:1.7.2",
```

Comment:
It looks like there are a couple places that use `com.google.auto.value:auto-value:1.7`, can you update that to 1.7.2 as well? And all of these places should probably use an `autovalue_version` variable to make sure they are using the same version.

Reply:
done.